editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -1447,55 +1464,25 @@ export var storyboard = (
 )
 `)
       })
-      it('cannot copy element that has code in its children array', async () => {
-        const editor = await renderTestEditorWithCode(
-          `import * as React from 'react'
-          import { Storyboard } from 'utopia-api'
-          
-          const width = 122
-          
-          export var storyboard = (
-            <Storyboard data-uid='sb'>
-              <div
-                style={{
-                  backgroundColor: '#aaaaaa33',
-                  position: 'absolute',
-                  left: 281,
-                  top: 329,
-                  width: 122,
-                  height: 73,
-                }}
-                data-uid='container'
-              >
-                <div
-                  style={{
-                    backgroundColor: '#aaaaaa33',
-                    position: 'absolute',
-                    left: 19,
-                    top: 19,
-                    width: width,
-                    height: 40,
-                  }}
-                  data-uid='child'
-                />
-              </div>
-            </Storyboard>
-          )
-          `,
-          'await-first-dom-report',
-        )
+      describe('repeated paste', () => {
+        async function pasteNTimes(editor: EditorRenderResult, n: number) {
+          const canvasRoot = editor.renderedDOM.getByTestId('canvas-root')
 
-        await selectComponentsForTest(editor, [EP.fromString('sb/container')])
+          for (const _ in Array(n).fill(0)) {
+            // paste 4 times
```

Comment:
This comment is a lie now :)

Reply:
fixed in `[7442e2f](https://github.com/concrete-utopia/utopia/pull/3798/commits/7442e2f0fd888dff214981287d11f3de11de9d28)`

Reply:
Hahaha ok technically it's not because you use 4 every time, but still!